data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/KafkaConsumerRecordUtils.java

```diff
@@ -31,7 +31,6 @@ public static <T> ConsumerRecord<T, CloudEvent> copyRecordAssigningValue(final C
       record.offset(),
       record.timestamp(),
       record.timestampType(),
-      record.checksum(),
```

Comment:
here we do already remove the `checksum`, since removed :-)

Reply:
yeah, here type inference works